bioio_base/test_utilities.py

```diff
@@ -66,8 +66,6 @@ def run_image_container_checks(
     expected_physical_pixel_sizes: Tuple[
         Optional[float], Optional[float], Optional[float]
     ],
-    expected_scale: Tuple[Optional[float]],
```

Comment:
This is the removal of time_interval and scale from the test utilities mention in the feature description.